### DISCUSSION TITLE
filtering RegEx

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#6086] [Unsupported Versions] [answered]
I see logs under kerberos.client, and I want to filter out anything that has a $ for example computer08$/domain.com. I just want to filter out anything that has that $ in the middle so i can see everything without it so i can see users/domain.com

**Comment by dougburks** [comment, marked as answer]
Instead of trying to filter the kerberos log, try the ntlm log instead. Here's a Hunt query:
```
event.dataset:ntlm | groupby user.name
```

For example:
![image](https://user-images.githubusercontent.com/1659467/140528028-811a3787-a0b9-4af2-9d09-fe9a96d06052.png)

**Reply by DinhDiesel** [reply]
@dougburks that does bring up all users, but i have computers that show up as comp013$. I'm trying to filter out everything with $ at the end...

**Reply by dougburks** [reply]
If `comp013` is based on your standard computer naming convention, then you might be able to do something like this:
```
event.dataset:ntlm AND NOT user.name:comp* | groupby user.name
```

**Reply by DinhDiesel** [reply]
@dougburks thank you!
